superset-websocket/eslint.config.js

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
NIT: would have been great to `git mv` the file instead of what I'm guessing was a simple `mv`, which would make the PR easier to review